modules/network-observability-RTT.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-RTT_{context}"]
+= Working with RTT tracing
+You can track RTT by editing the `FlowCollector` to the specifications in the following YAML example.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. In the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*.
+. Select *cluster*, and then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource for RTT tracing, for example:
++
+[id="network-observability-flowcollector-configuring-RTT_{context}"]
+.Example `FlowCollector` configuration
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: DIRECT
+  agent:
+    type: EBPF
+    ebpf:
+      features:
+       - FlowRTT               <1>
+----
+<1> You can start tracing RTT network flows by listing the `FlowRTT` parameter in the `spec.agent.ebpf.features` specification list.
+
+.Verification
+When you refresh the *Network Traffic* page, the *Overview*, *Traffic Flow*, and *Topology* views display new information about RTT:
+.. In the *Overview*, select new choices in *Manage panels* to choose which graphical visualizations of RTT to display.
+.. In the *Traffic flows* table, the *Flow RTT* column can be seen, and you can manage display in *Manage columns*.
+.. In the *Traffic Flows* view, you can also expand the side panel to view more information about RTT. 
+.. Click *Common* filters -> *Protocol*. Filter the network flow data based on *Protocol TCP* to show flows with nonzero values.
```

Comment:
We actually have some zeros in that case, even with sampling 1
https://issues.redhat.com/browse/NETOBSERV-1390

@msherif1234 is it worth putting a fancy technical limitation here ?
Or maybe just filter `"Flow RTT > 0"` as mentioned below is good enough ?

Reply:
I think this bug has been closed now, and the behavior is working as expected so I don't think there's anything additional to document here. Can you confirm @msherif1234 @jpinsonneau ?

Reply:
probably adding `Direction` = `Ingress` since this measurement was done for incoming TCP traffic

Reply:
Got it, thanks. I re-wrote this section as "Example filtering" to try within the procedure.